### DISCUSSION TITLE
Usage of Apollo-client-tools with Chrome and Nativescript

### DISCUSSION THREAD

**Discussion by hwillson** [apollographql/apollo-client-devtools#388] [Ideas]
Migrated from: https://github.com/apollographql/apollo-client-devtools/issues/269

Hello together, I am currently using the Apollo client within a nativescript application. Nativescript uses the chrome devtools for debugging. However, I am not able to use the apollo-client-devtools here, as they do not show up. I am wondering, if I might need to configure something in order to be able to open the apollo-client-devtools or if the tools are not working together with nativescript apps. Thanks for your help.

**Comment by cjohn001** [comment]
Hello @hwillson,
I assume the problem is that the nativescript runtime does not export the apollo cache structure. I have seen the following youtube video
    https://www.youtube.com/watch?v=ou0fEW1eRjc&t=17s 

Minute: 1:53.06 

Seems like the global object __APOLLO_CLIENT__ should be present in the debugger. It is undefined in the chrome dev tools when I debug a nativescript app. Does the plugin need other variables except __APOLLO_CLIENT__  in order to work? 
I would open a feature request on the nativescript github project than and ask if the guys could export the required environment variables.

Best regards,
 Christoph
